website/docs/Installation.md

```diff
@@ -31,6 +31,27 @@ The following command will deactivate the current `conda` environment:
 conda deactivate
 ```
 
+### Option 3: poetry
+
+Another option is with `poetry`, which is a dependency manager for Python. 
+
+[Poetry is a tool for dependency management and packaging in Python. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you. Poetry offers a lockfile to ensure repeatable installs, and can build your project for distribution.
+](https://python-poetry.org/docs/)
```

Comment:
```suggestion
[Poetry](https://python-poetry.org/docs/) is a tool for dependency management and packaging in Python. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you. Poetry offers a lockfile to ensure repeatable installs, and can build your project for distribution.
```

Reply:
Looks good to me :)

Reply:
Sorry, I don't get it. Would you like to accept the suggestion or not?

Reply:
Have hyper link on the whole paragraph with several sentences doesn't look nice.
![image](https://github.com/microsoft/autogen/assets/3197038/09928e98-65d6-4f1a-9fe9-f3bcb77ff27b)